extensions/wikia/FacebookClient/FacebookClientXFBML.php

```diff
@@ -48,12 +48,13 @@ public static function parserHook( $tag, $text, $args, Parser $parser ) {
 			unset( $args['profileid'] );
 		}
 
+		// add custom attribute to look for in DOM. We use this to determine
+		// if the Facebook SDK should be loaded.
+		$args['data-type'] = 'xfbml-tag';
```

Comment:
I couldn't find a good way to search for all the custom fb tags (eg `<fb:like>, <fb:share>, <fb:follow>'), so I'm adding a custom attribute to check for in the DOM.